apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -86,39 +86,6 @@ export const COLUMNS = {
   ACTIONS: 4,
 };
 
-// Cell formatters.
```

Comment:
For context on this diff since it looks a little weird:

The `passwordFormatter` function was previously outside of the `PasswordReset` component. Now that `passwordFormatter` needs access to `this.props.sectionId` from `PasswordReset`, I moved it into the component.